code_samples/logging_level_sample.js

```diff
@@ -17,11 +17,11 @@
 
 const { Client } = require('hazelcast-client');
 
-// This sample code demonstrates logging. We don't need to do any operations.
+// This sample code demonstrates configuring logging level. We don't need to do any operations.
 (async () => {
     const client = await Client.newHazelcastClient({
         properties: {
-            'hazelcast.logging.level': 'DEBUG'
+            'hazelcast.logging.level': 'DeBuG'
```

Comment:
Can we add a comment saying that this string is case-insensitive? Then it can be 'DEBUG' if you want. I meant to add a comment about it.

Reply:
I added a comment for this value.